src/Processors/QtiV2/Out/QuestionTypes/LongtextMapper.php

```diff
@@ -13,6 +14,15 @@ public function convert(BaseQuestionType $questionType, $interactionIdentifier,
     {
         /** @var longtext $question */
         $question = $questionType;
+        $questionData = $question->to_array();
+        
+		// $feedbackOptions is used to create feedback or modelfeedback elements in responseProcessing
```

Comment:
Can you please fix indentation for the comment?

Reply:
Done